assets/scss/main.scss

```diff
@@ -15,3 +15,6 @@
 // COMPONENT SPECIFIC
 //
 @import "popover";
+
+// TODO: get this imported at the component level
+@import '../../node_modules/flatpickr/dist/flatpickr.css';
```

Comment:
We should let webpack resolve imports from `node_modules`.

```
@import "~flatpicker/dist/flatpickr.css";
```

In the webpack `sass-loader`, `~moduleName` is an alias for the path to `node_modules`. This is important because the relative path to the node modules directory may not be the same in every repo.